songbird/queue.py

```diff
@@ -44,11 +44,12 @@ class Queue(list):
     def __init__(
         self,
         driver: Driver,
-        on_next: Callable[[Driver, Any], Awaitable[None]] = None,
-        on_fail: Callable[[Driver, Any], Awaitable[None]] = None
+        on_next: typing.Callable[[Driver, typing.Any], typing.Awaitable[None]] | None = None,
+        on_fail: typing.Callable[[Driver, typing.Any], typing.Awaitable[None]] | None = None
```

Comment:
Only from imports should be used in this project

Reply:
What's wrong?

Reply:
Its typing